www/class/centreon-partition/partEngine.class.php

```diff
@@ -606,18 +606,50 @@ public function backupParts($table, $db)
     /**
      *
      * Check if MySQL version is compatible with partitionning.
+     * @param $db The Db singleton
+     * @return boolean
      */
     public function isCompatible($db)
     {
-        $DBRESULT = $db->query("SELECT plugin_status FROM INFORMATION_SCHEMA.PLUGINS WHERE plugin_name = 'partition'");
-        $config = $DBRESULT->fetchRow();
-        $DBRESULT->closeCursor();
-        if ($config["plugin_status"] != "ACTIVE") {
-            return (false);
+        $dbResult = $db->query("SELECT plugin_status FROM INFORMATION_SCHEMA.PLUGINS WHERE plugin_name = 'partition'");
+        $config = $dbResult->fetch();
+        $dbResult->closeCursor();
+        if ($config["plugin_status"] == "ACTIVE") {
+            unset($config);
+
+            return true;
         }
```

Comment:
line

Reply:
are you talking about the line between the unset and the return ? If so, I removed it ;)